testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/hibernate/secondlevelcache/SFSB.java

```diff
@@ -70,9 +70,9 @@ public void setupConfig() {
         try {
 
             // prepare the configuration
-            Configuration configuration = new Configuration().setProperty(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS,
-                    "true");
+            Configuration configuration = new Configuration().setProperty(AvailableSettings.USE_NEW_ID_GENERATOR_MAPPINGS, "true");
             configuration.getProperties().put(AvailableSettings.JTA_PLATFORM, JBossAppServerJtaPlatform.class);
+            configuration.getProperties().put(AvailableSettings.TRANSACTION_COORDINATOR_STRATEGY, "jta");
```

Comment:
ORM 5.0.x added the TRANSACTION_COORDINATOR_STRATEGY="jta" setting requirement but it wasn't until later in ORM 5.1, that a bug fix (HHH-11024) required this to be set, that impacts schema drop code.  So, its not a new setting, as ORM 5.0.x also had the setting.